### DISCUSSION TITLE
consumers are intact with the server but the messages are not processes

### DISCUSSION THREAD

**Discussion by webSudarshan** [php-amqplib/php-amqplib#1115] [Q&A]
I am using the latest version of php-amqplib and rabbitmq-3.12.2, when there are no requests for some long period of time my messages are not being processed. I have tried sending heartbeats but it didn't work and there is no exceptions as well.

**Comment by lukebakken** [comment]
Hello, thanks for using this library. We need some more information -
* What version of PHP and this library are you using?
* Please share a _complete_ set of code that demonstrates this issue.
* What does RabbitMQ log when you expect messages to be processed but they are not?
